spec/jobs/assignment_repo/create_github_repository_job_spec.rb

```diff
@@ -347,7 +353,7 @@
 
       it "fails to save the AssignmentRepo and reports" do
         allow_any_instance_of(AssignmentRepo).to receive(:save!).and_raise(ActiveRecord::RecordInvalid)
-
+        expect(GitHubClassroom.statsd).to receive(:increment).with("exercise_repo.create.fail")
```

Comment:
@BenEmdon Does this look right to you? I'm not sure if both of these metrics should be getting hit, or just the v2 one.

Reply:
Even I had the same doubt, this is present in `GroupAssignmentRepo::Creator` too, should we stop sending stats for v1 in both? Also, if we're changing this, are any other places we can add more tracking?

Reply:
I had the `v2` prefixed metrics to compare changes I made last summer. I think its time to drop the `v2` prefix. 
If we do choose to do so we must make sure our data dog dashboards are pointed at the correct metrics ⚠️

Reply:
So should I just remove `v2` prefix on these stats? Or i could send both `v2` prefixed and not prefixed stat in this PR and then send a separate PR to remove `v2`. Just so that we don't lose any tracking. What do you think?

Reply:
> Or i could send both v2 prefixed and not prefixed stat in this PR and then send a separate PR to remove v2 prefix. Just so that we don't lose any tracking.

This ☝️ 🎉

Reply:
awesome! thanks! 🎉